crates/plex-api/src/server/transcode.rs

```diff
@@ -1023,3 +1085,368 @@ where
         _ => Err(crate::Error::from_response(response).await),
     }
 }
+
+#[derive(Deserialize)]
+#[serde(rename_all = "lowercase")]
+enum QueueStatus {
```

Comment:
I'm not sure about what I'll say, so please use your best judgment :) I'm not extremely comfortable with files longer than ≈700 lines. How do you feel about moving `transcode.rs` into its own mod-folder, and splitting it into a few files?

Not a requirement for this PR, of course.